configure.ac

```diff
@@ -1302,37 +1302,33 @@ VIPS_INCLUDES="$VIPS_INCLUDES $TIFF_INCLUDES"
 VIPS_LIBS="$VIPS_LIBS $TIFF_LIBS"
 
 # Look for libspng first
-# 0.6.1 uses "libspng.pc", git master libspng uses "spng.pc"
```

Comment:
It seems that we still have to search for `libspng.pc` when we build with CMake (which is done in OSS-Fuzz). See for example:
https://github.com/randy408/libspng/blob/957d1ec1c19247767e245ab708903048f1220db8/CMakeLists.txt#L62
https://github.com/google/oss-fuzz/blob/c34c308faad86d154b52586ff66de8d77187cafd/projects/libvips/build.sh#L113-L114

(noticed while inspecting the CIFuzz build logs)

Reply:
Ouch, OK, let's look for both. Thanks for pointing this out!

Reply:
The `PACKAGES_USED="$PACKAGES_USED libspng"` thing is also needed when `libspng.pc` is used, unfortunately. Otherwise, dependencies that link to `vips.pc` or `vips-cpp.pc` may fail (since it cannot find `spng.pc`).

Reply:
Ooop, good point. I added this too.